### DISCUSSION TITLE
Choosing the Best Communication Channel for 8vim: WhatsApp, Discord or Matrix?

### DISCUSSION THREAD

**Discussion by flide** [8VIM/8VIM#390] [Polls]
Hello 8vim community!

As some of you might recall, We previously used the Matrix network as our unofficial communication system. Unfortunately, I've lost access to it and, given the platform's limitations, can't recover the password. As the 8vim community continues to grow, I believe it's essential to have a reliable and accessible communication channel.

I'm considering setting up a dedicated channel and have narrowed down the options to WhatsApp (with its new community features) and Discord. Both platforms have their merits. Personally, I lean towards WhatsApp as it's my primary mode of communication, ensuring I stay connected with all our community discussions. However, your feedback is paramount, and I want to make a decision that benefits the majority.

Please vote below and share your thoughts!

**Comment by kjoetom** [comment]
Hi flide,

Both Whatsapp and Discord are proprietary. Maybe we should think about open source alternatives, ideally with a decentralized structure. 
Is there anything against staying on the Matrix network?
I have to admit that I haven't looked at the current software options for a while and can't make a well considered suggestion.

Well, that's just a thought of mine.

kjoe

**Reply by flide** [reply]
Well I am a privacy fanatic myself, and do believe that a decentralised distrust based end to end encrypted open source system would be great... but let's be honest, what is it exactly anyone has against whatsapp? it's probably gonna last longer than any open source system, because fb's got money, and it is end to end encrypted. Only thing against it is our idealism. 

I have about 5 chat systems installed on my phone and I check only one. I know what it says about me, and I don't want to fight the easy course without a very very good reason.

**Comment by kjoetom** [comment]
For me, this is not just about idealism. I simply don't want to accept that the only way to set up an account is to give a messaging service my private phone number.
On the other hand, if I don't want to do that, I don't have to, and I won't :-)
So, in fact, it's not a problem. I can see the potential advantages of a communication channel that many people already use.

**Reply by flide** [reply]
Discord doesn't require phone number, does it?

**Reply by MaethorNaur** [reply]
the phone number is not required if I'm not mistaken

**Comment by kjoetom** [comment]
Supposedly not, because the phone number is needed for anti-abuse purposes. At least, that's what it says in this brilliant Messenger overview maintained and offered by Mike Kuketz ([--> here](https://www.messenger-matrix.de/messenger-matrix-en.html)).

If you wanted to meet the following criteria (be open source; work without a phone number; be available for Android, iOS and web/desktop; be federated/decentralised; have a security audit), there would only be one messenger left anyway: Element (Matrix)

**Comment by MaethorNaur** [comment]
Issues, PR, merge should be notified on matrix now

**Comment by ManDay** [comment]
XMPP is also a good option to be mentioned, but I'm happy with Matrix for the moment. Strongly opposed to WhatsApp and (less so) Discord; and I wouln't call it "ideological", but more "by principle" (and also for some practical reasons). Sorry to hear about your account problems @flide , it would be great if this could be solved because a room with no moderator nor admin is just waiting for abuse! Perhaps in the worst case if support@matrix.org won't help you we could create a new room?
